lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationQuery.java

```diff
@@ -5,161 +5,171 @@
+   * @param domain or domains for filtering
    * @return the query
    */
   ClassificationQuery domainIn(String... domain);
 
   /**
-   * Add to your query if the Classification shall be valid in its domain.
+   * Adds to your query if the {@linkplain pro.taskana.classification.api.models.Classification
+   * Classification} shall be valid in its domain.
    *
    * @param validInDomain a simple flag showing if domain is valid
    * @return the query
    */
   ClassificationQuery validInDomainEquals(Boolean validInDomain);
 
   /**
-   * Add your created-Dates to your query.
+   * Adds your created-dates to your query.
    *
-   * @param createdIn the {@link TimeInterval} within which the searched-for classifications were
-   *     created.
+   * @param createdIn the {@linkplain TimeInterval} within which the searched-for {@linkplain
+   *     pro.taskana.classification.api.models.Classification Classifications} were created
    * @return the query
    */
   ClassificationQuery createdWithin(TimeInterval... createdIn);
 
   /**
-   * Add your modified-Dates to your query.
+   * Adds your modified-dates to your query.
    *
-   * @param modifiedIn the {@link TimeInterval} within which the searched-for classifications were
-   *     modified the last time.
+   * @param modifiedIn the {@linkplain TimeInterval} within which the searched-for {@linkplain
+   *     pro.taskana.classification.api.models.Classification Classifications} were modified the
+   *     last time
    * @return the query
    */
   ClassificationQuery modifiedWithin(TimeInterval... modifiedIn);
 
   /**
-   * Add your name to your query.
+   * Adds your name to your query.
    *
-   * @param nameIn as String
+   * @param nameIn as Strings
    * @return the query
    */
   ClassificationQuery nameIn(String... nameIn);
 
   /**
-   * Add your name to your query. It will be compared in SQL with an LIKE.
+   * Adds your name to your query.
    *
-   * @param nameLike as String
+   * <p>It will be compared in SQL with the LIKE operator.
+   *
+   * @param nameLike as Strings
    * @return the query
    */
   ClassificationQuery nameLike(String... nameLike);
 
   /**
-   * Add your description to your query. It will be compared in SQL with an LIKE. If you use a
-   * wildcard like % then it will be transmitted to the database.
+   * Adds your description to your query.
+   *
+   * <p>It will be compared in SQL with the LIKE operator. If you use a wildcard like % then it will
+   * be transmitted to the database.
    *
    * @param descriptionLike your description
    * @return the query
    */
   ClassificationQuery descriptionLike(String descriptionLike);
 
   /**
-   * Add your priority to your query.
+   * Adds your priority to your query.
    *
    * @param priorities as integers
    * @return the query
    */
   ClassificationQuery priorityIn(int... priorities);
 
   /**
-   * Add your serviceLevel to your query.
+   * Adds your serviceLevel to your query.
```

Comment:
/**
   * Selects only {@linkplain Classification Classifications} which have a {@linkplain Classification#getServiceLevel()} value
   * equal to any of the passed values.
   *
   * @param serviceLevelIn the values of interest
   * @return the query
   */
  ClassificationQuery serviceLevelIn(String... serviceLevelIn);